src/main/kotlin/org/malachite/estella/task/api/TaskController.kt

```diff
@@ -62,6 +65,19 @@ class TaskController(
     private fun isPasswordProvided(organizationId: String?, taskStageUuid: String?, devMail: String?, password: String?): Boolean =
             listOfNotNull(organizationId, taskStageUuid, devMail).isNotEmpty() && password != null
 
+    @CrossOrigin
+    @Transactional
+    @GetMapping("/forJobSeeker")
+    fun getTasksForJobSeeker(
+            @RequestParam("taskStage") taskStageUuid: String?
+    ): ResponseEntity<Any> {
+        if (taskStageUuid == null)
+            return ResponseEntity.badRequest().body(Message("taskStageUuid is required"))
+
+        val tasks: List<TaskDto> =
+                taskStageService.getTaskStage(taskStageUuid).tasksResult.map { it.task.toTaskDto() }
+        return ResponseEntity.ok(tasks)
+    }
```

Comment:
1. Zamiast zwrócić Message, rzuciłbym nowy wyjątek NotEnoughParams czy jakoś tak który mógłby mieć w sobie wiadomość którą chcemy zwrócić. Dzięki temu moglibyśmy dać poprawny zwracany typ w ResponseEntity
2. usunąłbym klamry i dał przez równa się a drugą ścieżkę obsłużył jako 
``` kotlin
else taskStageService
.getTaskStage(taskStageUuid)
.tasksResult.map { it.task.toTaskDto() }
.let{ResponseEntity.ok(it)}
```

Reply:
Znaczy ogólnie jeśli taskStage jest required, to wystarczy nie dać tutaj pytajnika - wtedy spring sam ogarnie, że nie dostał potrzebnego parametru i rzuci źle - at least I hope so